### PR TITLE
EWS: Added additional dashboard report url

### PR DESCRIPTION
**Pull request by kkrampa**
Mike wants to have redirect from EWS v1 to particular location in dashboard report so this view will be useful.

### CODE REVIEW

**Comment by kkrampa** [comment]
@czue Build fail doesn't seem to be related. Could you kick off build again?

**Comment by czue** [comment]
kicked. @benrudolph do you think this is related to bower cache?

```
======================================================================
ERROR: test_compress_offline (corehq.apps.style.tests.test_compress_command.TestDjangoCompressOffline)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/style/tests/test_compress_command.py", line 8, in test_compress_offline
    call_command('compress', force=True)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 115, in call_command
    return klass.execute(*args, **defaults)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/django/core/management/base.py", line 533, in handle
    return self.handle_noargs(**options)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/compressor/management/commands/compress.py", line 274, in handle_noargs
    self.compress(sys.stdout, **options)
  File "/home/travis/virtualenv/local/lib/python2.7/site-packages/compressor/management/commands/compress.py", line 231, in compress
    "%s" % (template.template_name, e))
CommandError: An error occured during rendering /home/travis/build/dimagi/commcare-hq/corehq/apps/dashboard/templates/dashboard/dashboard_new_user.html: 'jquery/dist/jquery.min.js' could not be found in the COMPRESS_ROOT '/home/travis/build/dimagi/commcare-hq/staticfiles' or with staticfiles.
```

**Comment by benrudolph** [comment]
Almost certainly. I'll revert that PR. Looking at it it doesn't seem to
actually cache

File: custom/ewsghana/views.py [hunk 1]
```diff
@@ -191,6 +194,27 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+class DashboardRedirectReportView(RedirectView):
```
Comment (biyeun) [inline custom/ewsghana/views.py]: yessss thank you for not using a function-based view here :heart:

**Comment by kkrampa** [comment]
@gcapalbo Could you merge this?

**Pull request merged**